datastore/admin/datastore_admin_index_get.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package samples
+
+// [START datastore_admin_index_get]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	admin "cloud.google.com/go/datastore/admin/apiv1"
+	adminpb "google.golang.org/genproto/googleapis/datastore/admin/v1"
+)
+
+// indexGet gets an index.
+func indexGet(w io.Writer, projectID, indexID string) error {
+	// projectID := "my-project-id"
+	// indexID := "my-index"
+	ctx := context.Background()
+	client, err := admin.NewDatastoreAdminClient(ctx)
+	if err != nil {
+		return fmt.Errorf("admin.NewDatastoreAdminClient: %v", err)
+	}
+	defer client.Close()
+
+	req := &adminpb.GetIndexRequest{
+		ProjectId: projectID,
+		IndexId:   indexID}
```

Comment:
```suggestion
		IndexId:   indexID,
	}
```

This way, if you add another value at the end, you only modify that single line, rather than both. Also makes it clear when scanning the code where the end of the declaration is.